Revert: fdisk: do not treat names ending in '0' as partitions


/dev/mtdblock0 is not whole disk
---

diff --git a/util-linux/fdisk.c b/util-linux/fdisk.c
--- a/util-linux/fdisk.c
+++ b/util-linux/fdisk.c
@@ -2841,8 +2841,7 @@ list_devs_in_proc_partititons(void)
 			continue;
 		for (s = ptname; *s; s++)
 			continue;
-		/* note: excluding '0': e.g. mmcblk0 is not a partition name! */
-		if (s[-1] >= '1' && s[-1] <= '9')
+		if (isdigit(s[-1]))
 			continue;
 		sprintf(devname, "/dev/%s", ptname);
 		open_list_and_close(devname, 0);
--
cgit v0.9.0.1-2-gef13
